lib/paged-sync.js

```diff
@@ -146,7 +146,7 @@ function getSyncPage (http, items, query, { paginate }) {
   return http.get('sync', createRequestConfig({ query: query }))
     .then((response) => {
       const data = response.data
-      items = items.concat(data.items)
+      items = items.concat(data.items || [])
       if (data.nextPageUrl) {
```

Comment:
I think the issue we are seeing in our application is that nothing is returned from this getSyncPage function, therefore row 74 crashes on trying to access property items of undefined

Which I think would be fixed by adding an else branch in this function, on line 165

Reply:
Hi @nfcampos,

thx for clarifying. 

My solution here ensures that `items` will always exist, but you're right that it doesn't cover the case when `response.data` has neither `nextPageUrl` or `nextSyncUrl`. I'm happy to add an extra else here returning an empty `items` array, but would your app expect it to contain a sync token too, or any other properties? Maybe you could tell me a bit more about the case in which the sync api request completes successfully but contains no data (if you know :D )?

Reply:
I think just an empty items array from `getSyncPage` would work, without any other properties. Then `getSyncPage` will return an empty array. WDYT?

Reply:
Yeah I think that makes sense
Sadly I dont have any more details from my app other than what I posted on the issue, not entirely sure what caused it

Reply:
All looks good for me now after the latest commit:)